kevm-pyk/src/kevm_pyk/kompile.py

```diff
@@ -61,6 +62,7 @@ def kevm_kompile(
     debug_build: bool = False,
     debug: bool = False,
     verbose: bool = False,
+    type_inference_mode: str | TypeInferenceMode | None = TypeInferenceMode.SIMPLESUB,
```

Comment:
Consider

```suggestion
    type_inference_mode: str | TypeInferenceMode | None = None,
```

Then handle the default within the function.